internal/api/core/describe.go

```diff
@@ -157,18 +193,26 @@ func DescribeRoutes(routes []Route) (*APIDescription, error) {
 			errs = errors.Join(errs, err)
 		}
 
+		outputBaseFields := make([]BaseFieldDescription, 0, len(output.Fields))
+
```

Comment:
I would remove this newline.
Then creating and filling the slice can be in the same visual block.